devito/types/dense.py

```diff
@@ -997,11 +1019,7 @@ def __halo_setup__(self, **kwargs):
                 halo = (left_points, right_points)
             else:
                 raise TypeError("`space_order` must be int or 3-tuple of ints")
-            base = [halo if i.is_Space else (0, 0) for i in self.dimensions]
-            # left-/right-staggering require extra points
-            extra = [(-i, 0) if i < 0 else (0, i) for i in self.staggered]
```

Comment:
why dropping this?

Reply:
Not needed anymore with FD hack removed. Everything now as the size fo the grid and FD radius can't be higher than space_order anymore.

Reply:
cool, OK